.github/workflows/CI.yml

```diff
@@ -20,54 +20,24 @@ env:
   RUSTUP_MAX_RETRIES: 10
 
 jobs:
-  build:
-    runs-on: ubuntu-latest
-    concurrency: 
-      group: ${{ github.workflow }}-${{ github.ref }}-build
-      cancel-in-progress: true
-    steps:
-      - name: Free Up Disk Space
-        run: |
-          echo "Freeing up disk space..."
-          echo "Disk space before cleanup:"
-          df -h
-          sudo apt-get clean
-          sudo rm -rf /usr/local/lib/android /usr/lib/jvm /usr/local/share/boost /opt/hostedtoolcache
-          docker system prune -a -f
-          echo "Disk space after cleanup:"
-          df -h
-
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Rust install
-        uses: dtolnay/rust-toolchain@stable
-
-      - name: Install Protoc
-        uses: arduino/setup-protoc@v3
-        with:
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Cache
-        uses: actions/cache@v4
-        with:
-          path: |
-            ~/.cargo/bin/
-            ~/.cargo/registry/index/
-            ~/.cargo/registry/cache/
-            ~/.cargo/git/db/
-            target/
-          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-
-      - name: Build
-        run: cargo build --all --tests 
 
   fmt:
     runs-on: ubuntu-latest
+    container:
+      image: ubuntu:20.04
     concurrency: 
       group: ${{ github.workflow }}-${{ github.ref }}-fmt
       cancel-in-progress: true
     steps:
+      - name: Install Dependencies
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          apt-get update
+          apt-get install -y tzdata
+          ln -fs /usr/share/zoneinfo/UTC /etc/localtime
+          dpkg-reconfigure -f noninteractive tzdata
```

Comment:
i assume these two steps are required for the subsequent package installs to work or else we could combine our `apt-get` calls into a single one-time invocation?

Reply:
You are right I can probably combine those

Reply:
Fixed